app/src/main/java/com/cornellappdev/android/eatery/WeeklyMenuFragment.java

```diff
@@ -0,0 +1,364 @@
+						breakfastText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+						lunchText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.blue));
+						dinnerText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+
+						mealType = MealType.LUNCH;
+						changeListAdapter(mealType, presenter.getSelectedDate());
+					}
+				});
+
+		dinnerText.setOnClickListener(
+				new View.OnClickListener() {
+					@Override
+					public void onClick(View view) {
+						breakfastText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+						lunchText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+						dinnerText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.blue));
+
+						mealType = MealType.DINNER;
+						changeListAdapter(mealType, presenter.getSelectedDate());
+					}
+				});
+		LocalDateTime currentTime = LocalDateTime.now();
+
+		if (currentTime.getHour() < 11) {
+			changeListAdapter(MealType.BREAKFAST, presenter.getSelectedDate());
+			mealType = MealType.BREAKFAST;
+			breakfastText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.blue));
+			lunchText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+			dinnerText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+		} else if (currentTime.getHour() < 16) {
+			changeListAdapter(MealType.LUNCH, presenter.getSelectedDate());
+			mealType = MealType.LUNCH;
+			breakfastText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+			lunchText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.blue));
+			dinnerText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+		} else if (currentTime.getHour() < 22) {
+			changeListAdapter(MealType.DINNER, presenter.getSelectedDate());
+			mealType = MealType.DINNER;
+			breakfastText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+			lunchText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+			dinnerText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.blue));
+		} else {
+			presenter.getDayInWeek(1);
+			changeListAdapter(MealType.BREAKFAST, presenter.getSelectedDate());
+			mealType = MealType.BREAKFAST;
+			breakfastText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.blue));
+			lunchText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+			dinnerText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
```

Comment:
you end up having the `BREAFKAST` case twice here, this should be avoided!

Reply:
the difference between this `BREAKFAST` case and the first one is that we add 24 hours to the `presenter` so it actually shows "tomorrow's" breakfast menu. I'm not sure that I know a better way to rearrange this logically.